packages/ibm-products/src/components/Datagrid/Datagrid.stories.js

```diff
@@ -496,7 +496,12 @@ export const BatchActions = () => {
     useStickyColumn
   );
 
-  return <Datagrid datagridState={{ ...datagridState }} />;
+  return (
+    <Datagrid
+      datagridState={{ ...datagridState }}
+      toolbarLabel="batch actions toolbar"
```

Comment:
Just a nit. This "label" will never be visible to the user. Should it be renamed to something more self-descriptive, like "toolbarAriaLabel"?

Reply:
fair enough. fixt